cfg/lynx-bll.cfg

```diff
@@ -1,7 +1,8 @@
 SYMBOLS {
     __STACKSIZE__:        type = weak, value = $0800; # 2k stack
     __STARTOFDIRECTORY__: type = weak, value = $00CB; # start just after loader
-    __BLOCKSIZE__:        type = weak, value = $0400; # cart block size
+    __BANK0BLOCKSIZE__:   type = weak, value = 1024; # bank 0 cart block size
```

Comment:
Could you please change the constants (back) to four-digit hex constants for consistency reasons. Thanks in advance for your patience with my nitpicking ;-)